docs/section-7/odometry-kinematics.md

```diff
@@ -0,0 +1,57 @@
+## What is Odometry?
+Odometry is the use of sensor information to estimate the robot's position on the field. Odometry works for both differential drivetrains (think arcade/tank drive) and swerve drivetrains (covered in a later section - drivetrains that can move forward, strafe, and rotate, perhaps at the same time).
+
+## WPILib Odometry Class
+Before we cover odometry, we first need to cover the `Translation2d`, `Rotation2d`, and `Pose2d` objects.
```

Comment:
Link to the corresponding API docs for each class you reference, the first time you reference it.